haystack/schema.py

```diff
@@ -276,11 +283,27 @@ def load_from_args(cls, component_type: str, **kwargs):
         :param component_type: name of the component class to load.
         :param kwargs: parameters to pass to the __init__() for the component. 
         """
-        if component_type not in cls.subclasses.keys():
-            raise Exception(f"Haystack component with the name '{component_type}' does not exist.")
-        instance = cls.subclasses[component_type](**kwargs)
+        subclass = cls.get_subclass(component_type)
+        instance = subclass(**kwargs)
         return instance
 
+    @classmethod
+    def load_from_pipeline_config(cls, pipeline_config, component_name):
+        if pipeline_config:
```

Comment:
Please document here better what this function is actually supposed to do. If I get it right, it's loading a single component after we already read a yaml into this object called pipeline_config